.github/workflows/ci-createturbo.yml

```diff
@@ -0,0 +1,57 @@
+name: CI JS
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches: ["main", "mk/add-to-ci"]
```

Comment:
trying to test this here https://github.com/mehulkar/turborepo/pull/1

Reply:
turns GH actions run without merging the upstream branch, so I guess I don't need the test PR!